runtime/src/weights/pallet_xcm.rs

```diff
@@ -71,7 +71,9 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for ZKVWeight<T> {
         //  Measured:  `0`
         //  Estimated: `0`
         // Minimum execution time: 18_446_744_073_709_551_000 picoseconds.
-        Weight::from_parts(18_446_744_073_709_551_000, 0)
+        Weight::from_parts(160_940_000, 6196)
```

Comment:
Was this already executed on the reference machine ?

Reply:
No, these were executed locally, and should be considered placeholders.
I believe we will want to run benchmarks on the reference machine once the XCM configuration will be settled for our intended use cases.